prime-router/src/main/kotlin/cli/tests/LoadTestSimulator.kt

```diff
@@ -287,9 +299,9 @@ abstract class LoadTestSimulator : CoolTest() {
             true
         )
         val spike = Simulation( // Simulate a sudden spike in sends all at once
-            "Spike: 100 Threads each submitting once.",
-            100,
-            1,
+            "Spike: 200 Threads each submitting 25 times.",
```

Comment:
Increased this test so that its represents a greater load than we're actually seeing in Prod at this time.